generator_gomod.go

```diff
@@ -32,7 +38,12 @@ func (s *Spec) HasGomods() bool {
 
 func withGomod(g *SourceGenerator, srcSt, worker llb.State, opts ...llb.ConstraintsOpt) func(llb.State) llb.State {
 	return func(in llb.State) llb.State {
-		workDir := "/work/src"
+		const (
+			fourKB           = 4 * (1 << 10)
```

Comment:
Why not just hardcode 4096 here? It's more obvious and we won't need to unnecessarily compute this every time a worker is ran for a path.

Reply:
It's a const so it'll be computed once at compile time, but 4096 is a lot clearer so I'll use that instead.

Reply:
done